arduino-ide-extension/src/node/core-service-impl.ts

```diff
@@ -112,6 +112,15 @@ export class CoreServiceImpl extends CoreClientAware implements CoreService {
             reject(UserAbortApplicationError());
             return;
           }
+
+          if (
+            ServiceError.isInvalidArgument(error) &&
+            error.details.includes('instance is no longer valid')
```

Comment:
What should happen when the selected language is, for example, German, and there is a translation for this error message for the CLI?

Reply:
I'm now extracting the error type from the error metadata 👍